templates/partials/clone_and_build_blazing.j2

```diff
@@ -0,0 +1,27 @@
+{# This partial clones the BlazingSQL repo and build and installs it into the rapids conda env. #}
+
+{# Install build prerequisites #}
+RUN source activate rapids \
+    && conda install -y \
```

Comment:
do you want to use `gpuci_conda_retry` here?

Reply:
Yes. I think I hit the same issue we saw in the benchmarking job where it wasn't using the right env, but I'll work on it since it should be using `gpuci_conda_retry`